tips/TIP-0048/openapi3-core.yaml

```diff
@@ -0,0 +1,3718 @@
+              properties: 
+                profitMarginExponent:
+                  type: integer
+                  description: ProfitMarginExponent is used for shift operation for calculation of profit margin.
+                bootstrappingDuration:
+                  type: integer
+                  description: The length in epochs of the bootstrapping phase.
+                manaShareCoefficient:
+                  type: string
+                  description: The coefficient used for calculation of initial rewards.
+                decayBalancingConstantExponent:
+                  type: integer
+                  description: DecayBalancingConstantExponent is the exponent used for calculation of the initial reward.
+                decayBalancingConstant:
+                  type: string
+                  description: An integer approximation calculated based on chosen DecayBalancingConstantExponent. 
+                poolCoefficientExponent:
+                  type: integer
+                  description: PoolCoefficientExponent is the exponent used for shifting operation in the pool rewards calculations.
+              required:
+                - profitMarginExponent
+                - bootstrappingDuration
+                - manaShareCoefficient
+                - decayBalancingConstantExponent
+                - decayBalancingConstant
+                - poolCoefficientExponent
+            targetCommitteeSize:
+              type: integer
+              description: The target size of the committee.
+            chainSwitchingTreshold:
+              type: integer
+              description: The number of heavier slots that a chain needs to be ahead of the current chain to be considered for switching.
+          required:
+            - version
+            - networkName
+            - bech32Hrp
+            - tokenSupply
+            - storageScoreParameters
+            - workScoreParameters
+            - manaParameters
+            - genesisSlot
+            - genesisUnixTimestamp
+            - slotDurationInSeconds
+            - slotsPerEpochExponent
+            - stakingUnbondingPeriod
+            - validationBlocksPerSlot
+            - punishmentEpochs
+            - livenessThresholdLowerBound
+            - livenessThresholdUpperBound
+            - minCommittableAge
+            - maxCommittableAge
+            - epochNearingThreshold
+            - congestionControlParameters
+            - versionSignalingParameters
+            - rewardsParameters
+            - targetCommitteeSize
+            - chainSwitchingTreshold
+      required:
+        - startEpoch
+        - parameters
+
+    ValidatorsResponse:
+      description: Returns a paginated list of all registered validators ready for the next epoch and indicates if they were active recently (are eligible for committee selection).
+      properties:
+        validators:
```

Comment:
iota-core returns `stakers`, what should be changed?

Reply:
Yeah, we should discuss the definition of validators and stakers.